### PR TITLE
fix(explorer): interpret hex as string (minimist)

### PR DESCRIPTION
**Pull request by karooolis**

### CODE REVIEW

**Review by holic — APPROVED** [review]

**Review by holic — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -58,5 +58,10 @@
   "engines": {
     "node": "^18.20.1",
     "pnpm": "^9.6.0"
+  },
+  "pnpm": {
+    "patchedDependencies": {
```
Comment (holic) [inline package.json:63]: ooc does this have to be in the workspace root for it to work? or could this be in `packages/explorer/package.json`

File: package.json [hunk 2]
```diff
@@ -58,5 +58,10 @@
...
+      "minimist@1.2.8": "patches/minimist@1.2.8.patch"
```
Comment (holic) [inline package.json:64]: might be nice to name this something like `patches/minimist@1.2.8-allow-hex.patch`

Comment (karooolis) [reply package.json:64]: updated in https://github.com/latticexyz/mud/pull/3105

File: package.json (hunk 1 above)
Comment (karooolis) [reply package.json:63]: looks like it's not possible to scope it down to a package, pnpm complains if I do so :/

**Pull request merged**